docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -12,16 +12,27 @@ Query management properties
 Configures the algorithm to organize the processing of all of the
 stages of a query. You can use the following execution policies:
 
-* ``phased`` schedules stages in a sequence to avoid blockages because of 
-  inter-stage dependencies. This policy maximizes cluster resource utilization 
+* ``phased`` schedules stages in a sequence to avoid blockages because of
```

Comment:
extract formatting changes to separate commit

Reply:
Turns out neither IDEA nor Visual Code let me save the file with the trailing spaces anymore.

Reply:
Had to resort to vim .. haha.. done now.